pages/user_profile_page.py

```diff
@@ -0,0 +1,41 @@
+from locators import LocatorsUserPage
+
+
+class UserPage:
+    """User page with user's Profile"""
+    def __init__(self, browser):
+        self.browser = browser
+        self.locators = LocatorsUserPage
+
+    def clear_data_textbox(self, key):
+        """Clears specified textbox"""
+        self.browser.clear_element(self.locators.user_fields[key])
+
+    def enter_data_textbox(self, key, value):
+        """Enters into the specific field data and returns True/False for valid/invalid data"""
+        self.clear_data_textbox(key)
+        self.browser.send_keys(self.locators.user_fields[key], value)
+        return True if self.check_validity_data(key) else False
+
+    def check_invalidity_data(self, key):
+        """Verifies if UI alerts about invalid data input"""
+        temp = self.browser.get_attr_value(self.locators.user_fields[key], 'class')
```

Comment:
Change temp name to more obvious, EVERYWHERE

Reply:
Ok, I will